lib/msf/core/opt_base.rb

```diff
@@ -196,6 +197,10 @@ def invalid_value_length?(value)
     #
     # A optional regex to validate the option value
     #
+    attr_accessor :conditions
+    #
+    # The list of potencial conditions
```

Comment:
```suggestion
    # The list of potential conditions
```

Reply:
Thank you for the comments, I've changed the suggested comments.